server/isomorphic-routing/index.js

```diff
@@ -40,13 +50,18 @@ function getEnhancedContext( req, res ) {
 	} );
 }
 
-function applyMiddlewares( context, ...middlewares ) {
-	const liftedMiddlewares = middlewares.map( middleware => next => middleware( context, next ) );
+function applyMiddlewares( context, expressNext, ...middlewares ) {
+	const liftedMiddlewares = middlewares.map( middleware => next => middleware( context, ( err ) => {
+		if ( err ) {
+			expressNext( err ); // Call express' next( err ) for error handling (and bail early from this route)
```

Comment:
What exactly happens here? Somehow comment does not help me - I just am not too familiar with Express :/